lib/json_schemer/format.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
+require 'ecma-re-validator'
```

Comment:
I'm curious—is it possible to autoload gems as well? Something like:

```ruby
autoload :EcmaReValidator, 'ecma-re-validator'
```

With `require` it seems like we could easily accidentally reintroduce upfront loading if a future change happens to reference `JSONSchemer::Format`.